src/coreclr/nativeaot/System.Private.CoreLib/src/System/Exception.NativeAot.cs

```diff
@@ -112,6 +116,10 @@ private static void AppendExceptionStackFrame(object exceptionObj, IntPtr IP, in
                 bool isFirstFrame = (flags & (int)RhEHFrameType.RH_EH_FIRST_FRAME) != 0;
                 bool isFirstRethrowFrame = (flags & (int)RhEHFrameType.RH_EH_FIRST_RETHROW_FRAME) != 0;
 
+                // track count for metrics
+                if(isFirstFrame && !isFirstRethrowFrame)
+                    s_exceptionCount++;
```

Comment:
FYI this isn't thread-safe, you can lose increments. Not sure if it matters, @jkotas for feedback on how precise we want to be here, vs the cost of a memory barrier.

Reply:
Also, open question on behavior for overflow. It seems like people might assume this value is monotonic.

Reply:
> Also, open question on behavior for overflow. It seems like people might assume this value is monotonic.

Mentioned in https://github.com/dotnet/runtime/issues/65989#issuecomment-1055018779 , but nothing was done about it.

Reply:
Created #85997 for the CoreCLR fix